cloudsql/sqlserver/database-sql/go.mod

```
@@ -0,0 +1,5 @@
+module sample-sqlserver
```

Please use the full module path. Will be something like:

```suggestion
module github.com/GoogleCloudPlatform/golang-samples/cloudsql/sqlserver/database-sql
```